servicedirectory/quickstart/quickstart.go

```diff
@@ -0,0 +1,117 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START servicedirectory_quickstart]
+
+// Sample quickstart is a basic program that uses Cloud Service Directory.
+package main
+
+import (
+	"context"
+	"fmt"
+
+	servicedirectory "cloud.google.com/go/servicedirectory/apiv1beta1"
+	sdpb "google.golang.org/genproto/googleapis/cloud/servicedirectory/v1beta1"
+)
+
+func main() {
+	projectId := "your-project-id"
+	location := "us-west1"
+	serviceId := "golang-quickstart-service"
+	namespaceId := "golang-quickstart-namespace"
+	endpointId := "golang-quickstart-endpoint"
+
+	ctx := context.Background()
+	// Create a registration client.
+	registry, err := servicedirectory.NewRegistrationClient(ctx)
+	if err != nil {
+		fmt.Printf("servicedirectory.NewRegistrationClient: %v", err)
```

Comment:
Can use `log.Fatalf` here. Then no need to return.

Reply:
done for all of the place where we were returning in quickstart